libminifi/include/capi/Plan.h

```diff
@@ -46,11 +48,50 @@
 #include "capi/cstructs.h"
 #include "capi/api.h"
 
+enum FailureStrategy { AS_IS, ROLLBACK };
+
+bool intToFailureStragey(int in, FailureStrategy *out);
```

Comment:
*spelling.
Is this something to be used in C? Perhaps we should avoid booleans? stdbool wasn't introduced until C99..

Reply:
Nope, it's hidden behind the C API, the API wraps it to int return value.